server/src/main/java/com/netcracker/domain/Message.java

```diff
@@ -0,0 +1,26 @@
+package com.netcracker.domain;
+
+import java.io.Serializable;
+
+public class Message implements Serializable{
+    private final MessageType messageType;
+    private final String data;
+
+    public Message(MessageType messageType) {
+        this.messageType = messageType;
+        data = null;
```

Comment:
&#x1F34E; **Issue**
It is kind of anti-pattern to keep null values inside your object and return nulls from method calls.
You should use "Default Object" pattern (which is empty string for a string) to avoid working with nulls.
Or you might want to throw Runtime exception if anyone tries to getData() from objects of specific types (although, it doesn't seem to be the best fit here).